packages/web/src/ssr/track/+onBeforeRender.tsx

```diff
@@ -31,11 +35,25 @@ export async function onBeforeRender(pageContext: PageContextServer) {
       throw new Error(discoveryRequestUrl)
     }
 
-    const json = await res.json()
+    const { data } = await res.json()
+    const [apiTrack] = data
+    // Include artwork in the track object
+    const track = {
+      ...makeTrack(apiTrack),
+      cover_art: apiTrack.artwork['1000x1000']
```

Comment:
Can use SquareSizes here for these. Were you not pulling that in for a reason?

Reply:
Oh yeah totally, no I just forgot :)